hackclub.com.yaml

```diff
@@ -2645,6 +2646,10 @@ som-mail-system:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+southhighhack:
+  - ttl: 600
+    type: CNAME
+    value: south-high-hack.vercel.app.
```

Comment:
🚨 For vercel sites, cname to cname.vercel-dns.com instead of your site's vercel.app url
```suggestion
    value: cname.vercel-dns.com.
```